modules/components/pages/outputs/cypher.adoc

```diff
@@ -0,0 +1,409 @@
+= cypher
+:type: output
+:status: experimental
+:categories: ["Services"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Writes a batch of messages to any graph database that supports the Neo4j or Bolt protocols.
+
+Introduced in version 4.37.0.
+
+[tabs]
+======
+Common::
++
+--
+```yml
+# Common configuration fields, showing default values
+output:
+  label: ""
+  cypher:
+    uri: neo4j://demo.neo4jlabs.com # No default (required)
+    cypher: 'MERGE (p:Person {name: $name})' # No default (required)
+    database_name: ""
+    args_mapping: root.name = this.displayName # No default (optional)
+    basic_auth:
+      enabled: false
+      username: ""
+      password: ""
+    batching:
+      count: 0
+      byte_size: 0
+      period: ""
+      check: ""
+    max_in_flight: 64
+```
+--
+Advanced::
++
+--
+```yml
+# All configuration fields, showing default values
+output:
+  label: ""
+  cypher:
+    uri: neo4j://demo.neo4jlabs.com # No default (required)
+    cypher: 'MERGE (p:Person {name: $name})' # No default (required)
+    database_name: ""
+    args_mapping: root.name = this.displayName # No default (optional)
+    basic_auth:
+      enabled: false
+      username: ""
+      password: ""
+      realm: ""
+    tls:
+      skip_cert_verify: false
+      enable_renegotiation: false
+      root_cas: ""
+      root_cas_file: ""
+      client_certs: []
+    batching:
+      count: 0
+      byte_size: 0
+      period: ""
+      check: ""
+      processors: [] # No default (optional)
+    max_in_flight: 64
+```
+--
+======
+
+== Examples
+
+[tabs]
+======
+Writes to Neo4j Aura::
++
+--
+This is an example of how to write data to Neo4j Aura.
+```yaml
+output:
+  cypher:
+    uri: neo4j+s://example.databases.neo4j.io
+    cypher: |
+      MERGE (product:Product {id: $id})
+        ON CREATE SET product.name = $product,
+                       product.title = $title,
+                       product.description = $description,
+    args_mapping: |
+      root = {}
+      root.id = this.product.id 
+      root.product = this.product.summary.name
+      root.title = this.product.summary.displayName
+      root.description = this.product.fullDescription
+    basic_auth:
+      enabled: true
+      username: "${NEO4J_USER}"
+      password: "${NEO4J_PASSWORD}"
+```
+--
+======
+
+== Fields
+
+=== `uri`
+
+The connection URI for your graphing database. For more information, see https://neo4j.com/docs/go-manual/current/connect-advanced/[Neo4j's documentation^].
```

Comment:
Do we only support the Neo4j and Bolt schemes?

Reply:
@rockwotj... Could you confirm?